euphonic/force_constants.py

```diff
@@ -852,13 +867,13 @@ def _dipole_correction_init(self, eta_scale=1.0):
         inv_dielectric = np.linalg.inv(dielectric)
         sqrt_pi = math.sqrt(math.pi)
 
-        # Calculate real/recip weighting
+        # Make guess for balance between real/recip terms
         abc_mag = np.linalg.norm(cell_vectors, axis=1)
         mean_abc_mag = np.prod(abc_mag)**(1.0/3)
-        eta = (sqrt_pi/mean_abc_mag)*n_atoms**(1.0/6)
-        # Use eta = lambda * |permittivity|**(1/6)
-        eta = eta*np.power(np.linalg.det(dielectric), 1.0/6)*eta_scale
-        eta_2 = eta**2
+        upper_lambda = (sqrt_pi/mean_abc_mag*n_atoms**(1.0/6)
+                        *np.linalg.det(dielectric)**(1.0/6))
+        upper_lambda *= dipole_parameter
+        lambda_2 = upper_lambda**2

```

Comment:
Now you have used upper_lambda as a name with no indication of how it is upper for eta. A comment may be useful

Reply:
I only used upper_lambda as[ lambda](https://docs.python.org/3/tutorial/controlflow.html#lambda-expressions) is a reserved keyword in Python. I can add a comment to explain this.